docs/examples/custom_operations/gpu_python_operator.ipynb

```diff
@@ -69,6 +69,21 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "⚠️**Warning**⚠️\n",
+    "<div class=\"alert alert-block alert-info\">\n",
```

Comment:
```suggestion
    "<div class=\"alert alert-block alert-warning\">\n",
```
The "info" block gets the same color as `.. note` in RST, not `.. warning`.
Also, I'd consider moving the title inside the box, e.g., but that's up to you.
```suggestion
  <div class="alert alert-block alert-warning">
  <h3>⚠️ Warning</3>
```

Reply:
OK. I moved to `alert-warning` and removed exclamation marks.

Reply:
This particular file stayed unchanged.
The exclamation marks were OK (although I'd prefer this one ⚠ over the one with yellow background). I was rather curious than critical. I thought that maybe some emoji was unintentionally replaced with an image or a special character.